util/util.go

```diff
@@ -4,12 +4,22 @@ package util
 
 import (
 	"encoding/binary"
+	"errors"
 	"fmt"
 	"io"
-	"os"
 	"time"
 )
 
+// ErrUnableToLockNow is used to indicate that a lock cannot be
+// immediately acquired.
+var ErrUnableToLockNow = errors.New("unable to acquire the lock at the moment")
```

Comment:
Then I can't use a generic error since the name of the file is given to the `CreateLockFile`. Also, the caller in this case is `FileStore.GetExclusiveLock()` which when the lock cannot be acquired because it is already locked returns `false`, but no error. So I would not even have a way to report the file name. Moreover, `Store.GetExclusiveLock()` will not necessarily be file related if/when we have other type of stores.